src/win/poll.c

```diff
@@ -34,7 +34,9 @@ static const GUID uv_msafd_provider_ids[UV_MSAFD_PROVIDER_COUNT] = {
   {0xf9eab0c0, 0x26d4, 0x11d0,
       {0xbb, 0xbf, 0x00, 0xaa, 0x00, 0x6c, 0x34, 0xe4}},
   {0x9fc48064, 0x7298, 0x43e4,
-      {0xb7, 0xbd, 0x18, 0x1f, 0x20, 0x89, 0x79, 0x2a}}
+      {0xb7, 0xbd, 0x18, 0x1f, 0x20, 0x89, 0x79, 0x2a}},
+  {0xa00943d9, 0x9c2e, 0x4633,
+      {0x9b, 0x59, 0x00, 0x57, 0xa3, 0x16, 0x09, 0x94}}
 };
```

Comment:
Could we add comments to these to inform future travelers to these parts. From the descriptions, I see:
https://docs.microsoft.com/en-us/windows/win32/api/winsock2/nf-winsock2-wsaenumprotocolsw

E70F1AA0-AB8B-11CF-8CA3-00805F48A192
MSAFD Tcpip [TCP/IP][UDP/IP][RAW/IP]

F9EAB0C0-26D4-11D0-BBBF-00AA006C34E4
MSAFD Tcpip [TCP/IPv6][UDP/IPv6][RAW/IPv6]

9FC48064-7298-43E4-B7BD-181F2089792A
MSAFD L2CAP [Bluetooth]
MSAFD RfComm [Bluetooth]

A00943D9-9C2E-4633-9B59-0057A3160994
AF_UNIX